webpack/scenes/ContentViews/Details/Filters/CVPackageGroupFilterContent.js

```diff
@@ -216,54 +216,54 @@ const CVPackageGroupFilterContent = ({
-                  }}
-                  selections={allAddedNotAdded[selectedIndex]}
-                  isOpen={selectOpen}
-                  isCheckboxSelectionBadgeHidden
-                >
-                  {allAddedNotAdded.map(item =>
-                    <SelectOption aria-label={item} key={item} value={item} />)}
-                </Select>
-              </SplitItem>
-              <SplitItem>
-                <Button isDisabled={!hasNotAddedSelected} onClick={bulkAdd} variant="secondary" aria-label="add_filter_rule">
-                  {__('Add filter rule')}
-                </Button>
-              </SplitItem>
-              <SplitItem>
-                <Dropdown
-                  toggle={<KebabToggle aria-label="bulk_actions" onToggle={toggleBulkAction} />}
-                  isOpen={bulkActionOpen}
-                  isPlain
-                  dropdownItems={[
-                    <DropdownItem aria-label="bulk_add" key="bulk_add" isDisabled={!hasNotAddedSelected} component="button" onClick={bulkAdd}>
-                      {__('Add')}
-                    </DropdownItem>,
-                    <DropdownItem aria-label="bulk_remove" key="bulk_remove" isDisabled={!hasAddedSelected} component="button" onClick={bulkRemove}>
-                      {__('Remove')}
-                    </DropdownItem>]
-                  }
-                />
-              </SplitItem>
-            </Split>
+              <Split hasGutter>
+                <SplitItem data-testid="allAddedNotAdded">
+                  <Select
+                    variant={SelectVariant.single}
+                    onToggle={setSelectOpen}
+                    onSelect={(_event, selection) => {
+                      setSelectedIndex(allAddedNotAdded.indexOf(selection));
+                      setSelectOpen(false);
+                    }}
+                    selections={allAddedNotAdded[selectedIndex]}
+                    isOpen={selectOpen}
+                    isCheckboxSelectionBadgeHidden
+                  >
+                    {allAddedNotAdded.map(item =>
+                      <SelectOption aria-label={item} key={item} value={item} />)}
+                  </Select>
+                </SplitItem>
+                <SplitItem>
+                  <Button isDisabled={!hasNotAddedSelected} onClick={bulkAdd} variant="secondary" aria-label="add_filter_rule">
+                    {__('Add filter rule')}
+                  </Button>
+                </SplitItem>
+                <SplitItem>
+                  <Dropdown
+                    toggle={<KebabToggle aria-label="bulk_actions" onToggle={toggleBulkAction} />}
+                    isOpen={bulkActionOpen}
+                    isPlain
+                    dropdownItems={[
+                      <DropdownItem aria-label="bulk_add" key="bulk_add" isDisabled={!hasNotAddedSelected} component="button" onClick={bulkAdd}>
+                        {__('Add')}
+                      </DropdownItem>,
```

Comment:
Do we remove Add from kebab menu here as per the review?

Reply:
Yeah, we probably should. 
IF you think of any other locations let me know